rust-peer/src/main.rs

```diff
@@ -323,6 +385,12 @@ fn create_swarm(
                 ..Default::default()
             },
         ),
+        request_response: request_response::Behaviour::new(
+            FileExchangeCodec(),
+            // TODO: support ProtocolSupport::Full
+            iter::once((FileExchangeProtocol(), ProtocolSupport::Outbound)),
```

Comment:
So this means, Rust peers can't serve these files because they will not accept the inbound protocol. Only the JS peers can serve them, correct?